src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -51,6 +51,11 @@ import { Form } from "react-bootstrap";
 import QuestionInput from "./QuestionInput";
 import EntranceInput from "./EntranceInput";
 import { ImageCollectionInput } from "components/molecules/ImageInput/ImageCollectionInput";
+import { UserStatusManager } from "components/molecules/UserStatusManager";
```

Comment:
side note. Isn't this component too big (just a little bit 🤔 )?

Reply:
Name-wise or the code inside? It's around 100 line of code and most of it is types and props.

Reply:
oo, sorry. I meant `DetailsForm.tsx` file. It has 1000+ lines. (couldn't comment it's first line, so commented the 1st changed line)
Of course, it's OOS of the PR, but still.. got surprised.

Reply:
Yeah, it's pretty huge. We should refactor this into smaller chuncks.

Reply:
I’m pretty sure that’s legacy admin v1 isn’t it? It’s huge and horrible, but also something that we can hopefully delete ‘soon’?

Reply:
Correct

Reply:
Definitely don’t worry about cleanup/refactor of it then. We’ll just watch it 🔥 soon enough. The bigger the better! 😛